src/frontend/src/btc/components/convert/BtcConvertFees.svelte

```diff
@@ -0,0 +1,55 @@
+<script lang="ts">
+	import { isNullish } from '@dfinity/utils';
+	import { getContext } from 'svelte';
+	import { BTC_CONVERT_FEE } from '$btc/constants/btc.constants';
+	import { UTXOS_FEE_CONTEXT_KEY, type UtxosFeeContext } from '$btc/stores/utxos-fee.store';
+	import { ckBtcMinterInfoStore } from '$icp/stores/ckbtc.store';
+	import ConvertFee from '$lib/components/convert/ConvertFee.svelte';
+	import { CONVERT_CONTEXT_KEY, type ConvertContext } from '$lib/stores/convert.store';
+	import { i18n } from '$lib/stores/i18n.store.js';
+	import type { OptionAmount } from '$lib/types/send';
+
+	export let sendAmount: OptionAmount;
+
+	const { sourceToken, sourceTokenExchangeRate, destinationToken } =
+		getContext<ConvertContext>(CONVERT_CONTEXT_KEY);
+
+	const { store: storeUtxosFeeData } = getContext<UtxosFeeContext>(UTXOS_FEE_CONTEXT_KEY);
+
+	let kytFee: bigint | undefined;
+	$: kytFee = $ckBtcMinterInfoStore?.[$destinationToken.id]?.data.kyt_fee;
+
+	let satoshisFee: bigint | undefined;
+	$: satoshisFee =
+		isNullish(sendAmount) || Number(sendAmount) === 0
+			? 0n
```

Comment:
It's correct to display 0 when amount is undefined?

Reply:
I think for the moment this should fine, but ideally we need to somehow show a small heads up to users that utxos fee can only be calculated for sendAmount > 0.